.github/workflows/assign_issue.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: "Assign issue to a team"
+
+on:
+    issues:
+        types: [opened, reopened]
+
+jobs:
+  auto_assign_issue:
+    runs-on: ubuntu-latest
+    container: ghcr.io/datadog/agent-issue-auto-assign:latest
+    permissions:
+        issues: write
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+      with:
+        ref: ${{ github.head_ref }}
```

Comment:
I think it does not exist on Issue open event. According to: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context.
This is probably fine to checkout on main?

Reply:
It was a bad copy/paste, thanks for the notification